zuul.d/jobs.yaml

```diff
@@ -696,8 +691,6 @@
     run: playbooks/cloud-provider-openstack-acceptance-test-csi-cinder/run.yaml
     post-run: playbooks/cloud-provider-openstack-acceptance-test-csi-cinder/post.yaml
     nodeset: ubuntu-xenial-citynetwork
-    secrets:
-      - citynetwork_credentials
```

Comment:
if secrets cannot be inherited, will this also failed ?
https://github.com/theopenlab/openlab-zuul-jobs/blob/master/playbooks/cloud-provider-openstack-acceptance-test-csi-cinder/run.yaml#L4

did you run with this patch before merging?

Reply:
oh,my fault.